app/src/main/java/com/concordium/wallet/onboarding/data/datasource/SharedPreferencesDataSourceImpl.kt

```diff
@@ -8,9 +8,14 @@ class SharedPreferencesDataSourceImpl(private val prefs: SharedPreferences) : Sh
 
     companion object {
         private const val TERMS_AND_CONDITIONS_VERSION_ACCEPTED_KEY = "TermsAndConditionsAccepted"
+        private const val SEED_PHRASE_KEY = "SeedPhrase"
     }
 
     override var termsAndConditionsVersionAccepted: String
         get() = prefs.getString(TERMS_AND_CONDITIONS_VERSION_ACCEPTED_KEY, String.Empty)!!
         set(value) = prefs.edit { putString(TERMS_AND_CONDITIONS_VERSION_ACCEPTED_KEY, value) }
+
+    override var seedPhrase: String
+        get() = prefs.getString(SEED_PHRASE_KEY, String.Empty)!!
+        set(value) = prefs.edit { putString(SEED_PHRASE_KEY, value) }
```

Comment:
Is everything in the shared preferences automatically encrypted? Where can I see how the encryption is done?